csstest.js

```diff
@@ -431,6 +431,8 @@ function runTests(filter = '') {
 		// Filter list of specifications
 		if (filter === 'stable' && Specs[spec]?.status?.stability !== 'stable') {
 			continue;
+		} else if (filter === 'experimental' && Specs[spec]?.status?.stability === 'stable') {
```

Comment:
This syntax not works on older browsers.

Reply:
Optional chaining is already [supported in more than 90% of browsers according to caniuse](https://caniuse.com/mdn-javascript_operators_optional_chaining). So that choice was deliberate. Though I can replace it by old syntax.

Reply:
I've removed the optional chaining now.